docs/job-scheduling.md

```diff
@@ -92,6 +96,8 @@ In standalone mode, simply start your workers with `spark.shuffle.service.enable
 
 In YARN mode, follow the instructions [here](running-on-yarn.html#configuring-the-external-shuffle-service).
 
+In K8S mode, follow the future work [here](running-on-kubernetes.html#future-work).
```

Comment:
This looks wrong to me because there is actually nothing to **follow** in the future work section. May I ask what do you want to say here?